fluidplayer.js

```diff
@@ -3151,6 +3151,16 @@ var fluidPlayerClass = {
                                     h: parseInt(tempThumbnailCoordinates[3])
                                 });
                             }
+                        } else {
```

Comment:
I am pretty sure there is a way to make it look more compact instead of adding the new else clause with push, which by the way sets image a bit differently.